src/dashboard/components/ArticleType.tsx

```diff
@@ -92,7 +92,7 @@ const ArticleType = ({ loading, onCancel, onConfirm }: Props): JSX.Element => {
             <h1 className="typography__heading typography__heading--h1">{t('heading')}</h1>
             <div className="article-type__content">
                 <SelectField
-                    labelText="Choose and article type"
+                    labelText="Choose an article type"
```

Comment:
Should this use translations/localisation?

Reply:
good point ill fix that now